src/components/body/footer/Footer.tsx

```diff
@@ -1,15 +1,27 @@
 import {Link} from "react-router-dom";
 import LANGUAGES from '../../translations/constants';
 import {FormattedMessage } from 'react-intl';
+import { useTheme } from "../hooks/useTheme";
 
 type FooterProps = {
   onLanguageSwitch: (arg: string) => void
 }
 const Footer = (props: FooterProps) => {
-  const { onLanguageSwitch } = props
+  const dark="dark";
+  const light="light";
```

Comment:
Group the const as you will need to import them in other files:

export const THEME = {
DARK: 'dark',
LIGHT: 'light',
}